rustup-init.sh

```diff
@@ -31,7 +31,7 @@ RUSTUP_UPDATE_ROOT="${RUSTUP_UPDATE_ROOT:-https://static.rust-lang.org/rustup}"
 # NOTICE: If you change anything here, please make the same changes in setup_mode.rs
 usage() {
     cat <<EOF
-rustup-init 1.26.0 (577bf51ae 2023-04-05)
+rustup-init 1.27.0 (5623b7424 2024-01-29)
```

Comment:
Presumably this is generated, so belongs in the previous commit? Or did you have to change this manually?

Reply:
As per https://github.com/rust-lang/rustup/pull/3653#issuecomment-1918457423 I have to change this manually every time, unfortunately. I don't like this myself either, but is there another way?

Reply:
The pain point here is that we don't know in advance what the commit shasum will be (and this shasum might be changed with the merge queue, we might need an override somehow just for releasing or we should change the workflow).
The output of the script seems to be intentionally reflecting `rustup-init.exe` but the shasum will be always off-by-one as explained in the aforementioned comment.

Reply:
I did some further research and I haven't found an ideal way of doing this automatically. I guess we should stick to the original plan then. First I'll try a fast-forwarding merge to bypass the merge queue. If that is disallowed in out repo, I'll issue another PR to fix up the commit shasum.